app/models/equipment_model.rb

```diff
@@ -166,6 +166,14 @@ def document_attributes=(document_attributes)
     end
   end
 
+  def active_reservations
+    if AppConfig.first.requests_affect_availability
+      reservations.not_overdue.active_or_requested
+    else
+      reservations.not_overdue.active
+    end
+  end
+
```

Comment:
going to rethink this and move the logic to the active scope because the appconfig change should also affect the cart validations